File: Difficulty: Basic/For Loop - 2/for-loop-2.java
//{ Driver Code Starts

// } Driver Code Ends

import java.util.*;

public class Main {
    public static void main(String args[]) {
        // Create a Scanner object to read input from the user
        Scanner scanner = new Scanner(System.in);
        
        // Read the input string
        String inputString = scanner.nextLine();
        
        // Iterate through the string, printing every alternate character
        for (int index = 0; index <= inputString.length(); index += 2) {
            System.out.print(inputString.charAt(index));
        }
    }
}


//{ Driver Code Starts.
// } Driver Code Ends

File: Difficulty: Basic/For Loop - 2/for-loop-2.py
#{ 
 # Driver Code Starts
#Initial Template for Python 3

# } Driver Code Ends

# User function Template for Python3

def utility(s):
    # Loop through the string, taking every second character
    for i in range(0, len(s), 2):
        # Print the character without a newline after each print
        print(s[i], end='')


#{ 
 # Driver Code Starts.



def main():
    testcases=int(input()) #testcases
    while(testcases>0):
        s=input().strip()
        utility(s)
        print()##separating testcases outputs by newlines
        testcases-=1
        


        print("~")
if __name__=='__main__':
    main()
# } Driver Code Ends